docs/responsiveness/issue-management.md

```diff
@@ -0,0 +1,54 @@
+# Issue Management Guide
+
+This document outlines some of the key workflows and policies that dotnet/runtime maintainers use to triage issues and manage their backlogs. The purpose of this guide is to:
+
+1. Provide reference documentation to area owners.
+2. Improve transparency on how we handle issues with the wider community.
+3. Set expectations regarding automated issue management policies.
+
+## Issue Triage
+
+Issue triage is the initial phase of the issue management process in which feature area owners
+
+1. Classify the issue: is it a feature request? bug report? user question? is the issue unclear?
+2. Determine viability of the issue: is it likely that we would support the feature request in principle? does the bug report reproduce? is the reported issue by-design behavior? is the issue a duplicate?
+3. Estimate required effort: does the feature design need further refinement? how much work is needed to get the implementation done?
+4. Assign priority and milestones.
+
+An issue is defined as "triaged" if it either gets closed _or_ gets assigned to a milestone. Issue triage is typically a multi-step process that is performed collaboratively by the feature area owners, issue author and supported by our bot infrastructure. While most issues are typically triaged within a day or two of being reported, in some cases it might take longer depending on a variety of circumstances.
+
+### Triage Workflow
+
+1. The issue labeling bot assigns the `untriaged` label and infers an area label for the new issue.
```

Comment:
this doc is specific for dotnet/runtime, right? it's in this repo just as an example.
assuming so, this should probably explain area labels, and link to the area list.
there are also os- and arch- labels that inform ownership.

Reply:
@eiriktsarpalis, I have `Labels` section and link to the area list in the text from my repo.
`-- List of runtime areas and owners can be found in https://github.com/dotnet/runtime/blob/main/docs/area-owners.md.`

> this should probably explain area labels, and link to the area list.
> there are also os- and arch- labels that inform ownership.

Reply:
> this doc is specific for dotnet/runtime, right? it's in this repo just as an example.

The idea is to author an unofficial, experimental and incomplete issue management guide specifically to drive our experiments in dotnet/runtime. Any automated actions conducted as part of the experiments would link to the guide in this repo.